### ISSUE TITLE
Session generated in the browser (React) doesn't work in Node.

### ISSUE THREAD

**Issue by nhsaraiva** [gram-js/gramjs#574]
The situation is as follows:

1. Login is performed in the browser in a React application.
2. I save the session string to use it later.
3. When I need to use it in the backend (Node), the session doesn't start at all.

I tested the opposite scenario, generating a session through input in Node and passing it to the browser afterward, and this test worked. However, the reverse seems to have some issue.

Do I need to make any adjustments to the backend session to make it work with the session generated by the browser?

**Comment by painor** [comment]
Yes you need to change the IP in the session. The react session will have the websocket as ip and for the node one you would need telegram's ip.

**Comment by milukove** [comment]
@painor 
> Yes you need to change the IP in the session. The react session will have the websocket as ip and for the node one you would need telegram's ip.

Can you, please, tell a bit more how to do that? How to change the IP? Can't find anything about that in the docs.

**Comment by Pyth0nHater** [comment]
> [@painor](https://github.com/painor)
> 
> > Yes you need to change the IP in the session. The react session will have the websocket as ip and for the node one you would need telegram's ip.
> 
> Can you, please, tell a bit more how to do that? How to change the IP? Can't find anything about that in the docs.

Have you solved this problem?

**Comment by Pyth0nHater** [comment]
> Yes you need to change the IP in the session. The react session will have the websocket as ip and for the node one you would need telegram's ip.

can u send code example?
